tango_simlib/tango_sim_generator.py

```diff
@@ -8,35 +8,27 @@
 Simlib library generic simulator generator utility to be used to generate an actual
 TANGO device that exhibits the behaviour defined in the data description file.
 """
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
 
 import argparse
 import logging
 import os
 import time
 import weakref
+from builtins import map, object, range
 from functools import partial
 
 import numpy as np
-from tango import (
-    Attr,
-    AttrDataFormat,
-    AttrQuality,
-    AttrWriteType,
-    CmdArgType,
-    DevState,
-    UserDefaultAttrProp,
-)
+from future import standard_library
+standard_library.install_aliases()
+from future.utils import with_metaclass
```

Comment:
Fix imports as per [wiki](https://github.com/ska-sa/katdeveloper/wiki/Investigate-a-way-to-improve-Py3-migration-feedback-loop-on-tox#futurize-stage-2)

Reply:
Why is this marked resolved?